modules/tinymce/src/core/test/ts/browser/caret/FakeCaretImageCaptionTest.ts

```diff
@@ -0,0 +1,37 @@
+import { Waiter } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import ImagePlugin from 'tinymce/plugins/image/Plugin';
+
+describe('browser.tinymce.core.FakeCaretImageCaptionTest', () => {
+  const browser = PlatformDetection.detect().browser;
+  const isFirefox = browser.isFirefox();
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    plugins: 'image',
+    image_caption: true
+  }, [ ImagePlugin ]);
+
+  // skiped on FireFox since `setRawSelection` seems not to work on it
+  (isFirefox ? it.skip : it)('TINY-11997: should hide after tabbing inside figcaption', async () => {
+    const editor = hook.editor();
```

Comment:
Another version of this would be:

```suggestion
  
  it('TINY-11997: should hide after tabbing inside figcaption', async function () => {
    // skiped on FireFox since `setRawSelection` seems not to work on it
    if(isFirefox) { this.skip(); }
    const editor = hook.editor();
```

But I don't know if it's more readable or not

Reply:
yes, it's better in your way, reafactored